gnomad/assessment/validity_checks.py

```diff
@@ -1211,3 +1211,102 @@ def count_vep_annotated_variants_per_interval(
     )
 
     return interval_ht
+
+
+def check_missingness_of_struct(
+    struct_expr: hl.expr.StructExpression, prefix: str = ""
+) -> Dict[str, Any]:
+    """
+    Recursively check the fraction of missing values of all fields within a StructExpression.
+
+    Either a standalone or nested Struct can be provided.
+
+    :param struct_expr: A StructExpression to check for missing values.
```

Comment:
apologies for missing this before, but a details question: does the team typically format these docstring descriptions with "A" or ignore it? as in, "A StructExpression to check for missing values" vs "StructExpression to check for missing values"